MobileApp/components/NotificationList.js

```diff
@@ -3,30 +3,31 @@ import { Alert, View, StyleSheet, Text } from "react-native";
 import { List, Card, Avatar, Modal } from "react-native-paper";
 import { ScrollView } from "react-native";
 import { Ionicons, MaterialIcons } from "@expo/vector-icons";
-import { listStudents} from "../graphql/queries";
+import { listStudents,listInstitutions,createStudent} from "../graphql/queries";
 import{Auth,API} from "aws-amplify"
 
-const NotificationList = () => {
+const NotificationList = ({navigation}) => {
   const [expanded1, setExpanded1] = useState(false);
   const [expanded2, setExpanded2] = useState(false);
   const [expanded3, setExpanded3] = useState(true);
   const[student,setStudent]=useState(null)
   const[announcements,setAnnouncements]=useState([])
+  const[reminders,setReminders]=useState([])
+  const[dueDates,setDueDates]=useState([])
   const handlePress1 = () => setExpanded1(!expanded1);
   const handlePress2 = () => setExpanded2(!expanded2);
   const handlePress3 = () => setExpanded3(!expanded3);
 
   const showFullMessage = (key) => {
     Alert.alert(key.description);
   };
-
+  
   const fetchAnnouncements = async() => {
-      try{
-        let error="There appears to be a network issue.Please try again later"
+    let error="There appears to ne a network error. Please try again"  
```

Comment:
spelling error on `There appears to <ne> a network error. Please try again`

Reply:
Thank you for pointing that out. This error has been corrected